.buildkite/commands/shared-set-up-distribution.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -eu
+
+echo "--- :beer: Installing Homebrew Dependencies"
+# Sentry CLI needs to be up-to-date
+brew upgrade sentry-cli
+
+brew tap FelixHerrmann/tap
+brew install swift-package-list
+
+brew install imagemagick
+brew install ghostscript
```

Comment:
I need to double check, but I think we can remove these. They would have been used to updated the app icon in the installable builds, but I couldn't find the code for it when I looked.